gadget/gadget.go

```diff
@@ -459,8 +459,13 @@ func InfoFromGadgetYaml(gadgetYaml []byte, model Model) (*Info, error) {
 			// pass
 		case "grub", "u-boot", "android-boot", "lk":
 			bootloadersFound += 1
+		case "piboot":
+			if model.Grade() == asserts.ModelGradeUnset {
+				return nil, errors.New("piboot bootloader valid only for core20 onwards")
```

Comment:
core20 is the base, we typically use UC20 in error messages to refer to whole UC version